src/extension.ts

```diff
@@ -1,318 +1,388 @@
+      )?.split("/");
+      console.log(config);
+      if (config) {
+        repo = config[4].split(".")[0];
+        owner = config[3];
+      }
+      localUser = await gitAPI?.repositories[0]?.getGlobalConfig("user.name");
+
+      getPRsPerSHAs();
+      watermelonPanel.createOrShow(context.extensionUri);
+    })
+  );
+  vscode.window.onDidChangeTextEditorSelection((selection) => {
+    startLine = selection.selections[0].start.line;
+    endLine = selection.selections[0].end.line;
+    getSHAs();
+  });
+
+  if (vscode.window.registerWebviewPanelSerializer) {
+    // Make sure we register a serializer in activation event
+    vscode.window.registerWebviewPanelSerializer(watermelonPanel.viewType, {
+      async deserializeWebviewPanel(
+        webviewPanel: vscode.WebviewPanel,
+        state: any
+      ) {
+        // Reset the webview options so we use latest uri for `localResourceRoots`.
+        webviewPanel.webview.options = getWebviewOptions(context.extensionUri);
+        watermelonPanel.revive(webviewPanel, context.extensionUri);
+      },
+    });
+  }
 }
-function escapeFilePath(path:string| undefined){
-	if(path){
-		// $& means the whole matched string
-		return path.replace(/[. *\s+\ ?^${}()|[\]\\]/g, '\\$&');
-	}
-	else {return "";}
+function escapeFilePath(path: string | undefined) {
+  if (path) {
+    // $& means the whole matched string
+    return path.replace(/[. *\s+\ ?^${}()|[\]\\]/g, "\\$&");
+  } else {
+    return "";
+  }
 }
-
-async function getPRsPerSHAs(){
-
-	let owner = "";
-	// get repo name and owner basename
-	await exec(`cd ${escapeFilePath(folderRoute)} \n git config --get remote.origin.url`,
-		async function (error:string, stdout:string, stderr:string) {
-			const splitStdout = stdout.split("/");
-			// Retrieving the owner here is very important because it evades pulling issues from other repos
-			let localowner = splitStdout[3];
-			owner = localowner;
-			
-			// Get issues by commit hash and owner
-			await octokit.request(`GET /search/issues?type=Commits`, {
-				org: owner,
-				q: `hash:${arrayOfSHAs[0]}`
-			}).then(octoresp => {
-				const issuesBySHAs = octoresp.data.items;
-
-				// Show toast if no search results are found
-				if (issuesBySHAs.length === 0) {
-					vscode.window.showErrorMessage("No search results. Try selecting a bigger piece of code or another file.");
-				} else {
-					issuesBySHAs.forEach((issue: { url: any; }) => {
-						const issueUrl = issue.url;
-	
-						octokit.request(`GET ${issueUrl}/comments`).then(octoresp => {
-							// this paints the panel
-							watermelonPanel.currentPanel?.doRefactor({ command: "prs", data: octoresp.data})
-							//@ts-ignore
-						}).catch(err => {
-							console.log("octoerr: ", err);
-						});
-					})
-				}
-			}).catch(error=> console.log("octoERR", error))
-
-		}
-	);
+function getPRsPerSHAs() {
+  octokit
+    .request(`GET /search/issues?type=Commits`, {
+      q: `hash:${arrayOfSHAs[0]}`,
```

Comment:
You're missing `org: owner,` so we don't get random pr comments from random repos in random organizations

Reply:
this isn't in the dev branch yet